ng2c3.component.ts.ts

```diff
@@ -0,0 +1,80 @@
+import {Component, Input, SimpleChange, ElementRef, ViewEncapsulation} from '@angular/core';
+import * as c3 from 'c3';
```

Comment:
I can't quite understand which part you are referring to! Are you saying about the inclusion of 'c3'?